meshapp/src/pages/Profiles/Page.tsx

```diff
@@ -261,20 +251,23 @@ const ProfileRole = (props: { isMentor: boolean; isMentee: boolean }) => {
 // TODO: To be fully implemented
 const ProfileExperience = (props: any) => {
   return (
-    <ThemeProvider theme={theme}>
-      <Box className="profile-page-column-body">
-        <Typography
-          variant="h1"
-          sx={{
-            marginBottom: "20px",
-            fontSize: "22px",
-          }}
-        >
-          Experience
-        </Typography>
-        <TestComponent />
-      </Box>
-    </ThemeProvider>
+    // <ThemeProvider theme={theme}>
```

Comment:
why are there comments here?

Reply:
That's my bad I meant to remove that. And as for your question on why ThemeProviders were there and are being removed now, there is actually a global ThemeProvider and Theme being applied to the entire app currently; I think at the time Patrick was developing this this wasn't implemented yet so he made his own Theme and ThemeProviders to initially style them. Now that there is I removed them so I could properly adapt them to the global Theme.

Reply:
ah okay yea a global theme is what I expect, thank you for the explanation, i was a bit confused.